pkg/migration/runner.go

```diff
@@ -670,8 +678,7 @@ func (r *Runner) createCheckpointTable(ctx context.Context) error {
 	checksum_watermark TEXT,
 	binlog_name VARCHAR(255),
 	binlog_pos INT,
-	rows_copied BIGINT,
-	alter_statement TEXT
```

Comment:
The rows_copied can not apply to multi-migrations because we need to know a number per chunker, so this is pushed into the watermark.

The optimistic chunker does not need this number, since it can use the value based on the current chunkPtr. The composite chunker does need this, so it now writes that into its checkpoint. The checkpoints are technically different for all 3 chunkers as a result.